gitSpy/USER_KEYS.js

```diff
@@ -1,4 +1,4 @@
-// INSERT YOUR OWN KEYS ^_^
+// INSERT YOUR OWN KEYS WITHOUT THE QUOTES ^_^
```

Comment:
@Rhombus33 Consider doing <YOUR USER_ID HERE> instead of "YOUR USER_ID HERE", if you don't want the quotes. I think it's easy to overlook the "without the quotes" instruction, and the <carrot brackets> do a better job of implying you should replace the whole thing.